enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -114,6 +108,8 @@ scrape_configs:
     kubernetes_sd_configs:
       - role: endpoints
 
+    honor_labels: true
```

Comment:
honor_labels makes https://github.com/lightbend/es-backend/issues/257 more important - to avoid conflicts.

Reply:
Conflicts on these labels-- workload and monitor_type -- are expected and when we see them we always want to keep the original.

Am seeing some of this as well in the `namespace` label which is why I think some things were labeling `kubernetes_namespace` as a way of sidestepping that conflict.  I'd rather just use one label for namespace and honor_labels.

Reply:
Not sure what you mean that they are expected. What if a customer already uses `monitor_type` for their own purposes? We can't ask them to change their app right? That's why I thought https://github.com/lightbend/es-backend/issues/257 is necessary.

Reply:
Well that would definitely be a bummer, and indeed the point of the other issue.  No this is the design of intentionally setting monitor_type upstream, to adopt a set of monitors for a workload.  For example, you can cook up a set of monitors with
```
monitor_type="my_special_akka_monitors"
```
and then use those to monitor your deployment by using that as an annotation.

Reply:
https://github.com/lightbend/es-backend/issues/279 to document this